### DISCUSSION TITLE
Can Affine display a public website similar to Notion sites?

### DISCUSSION THREAD

**Discussion by nordseo** [toeverything/AFFiNE#8415] [General]
Just stumbled on Affine. Looks like an excellent Notion replacement with the attractive mindmaps. But, I wanted to ask first if Affine can also be used to display a website like Notion sites.

**Comment by codes-from-nate** [comment]
I have the same question! I like that Notion allows users to make some pages public while others remain private. Can Affine do the same thing?

**Comment by CptCurk** [comment]
I would love that too. Thank you !
